.github/workflows/autofix.yml

```diff
@@ -0,0 +1,24 @@
+name: Run Autofix
+on:
+  schedule:
+    - cron:  '0 2 * * MON-FRI' # every workday at 2AM
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: "18.x"
+
+    - name: Run autofix
+      run: |
+        npm -g autofix
```

Comment:
nit: would be cool to tie versions here somehow, so we have at least some amount of control over what code we pass our GITHUB_TOKEN to... :confused: Can we maybe hard-code a specific version of autofix at least?

But I guess we are already doing this somewhere else, and this is merely a better way to execute it, so partly out of scope here. :confused:

Reply:
@geropl thanks for pointing this out! I will keep Autofix updated and have now pinned the version to the latest one 🙏